src/main/java/org/folio/service/export/storage/AmazonFactory.java

```diff
@@ -0,0 +1,22 @@
+package org.folio.service.export.storage;
+
+import org.springframework.stereotype.Component;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.transfer.TransferManager;
+import com.amazonaws.services.s3.transfer.TransferManagerBuilder;
+
+@Component
+public class AmazonFactory {
+  private Regions region = Regions.DEFAULT_REGION;
```

Comment:
this shouldn't be default region, please remove it. The region will be pulled from the environment as well.

Reply:
@KVupp, I agree. Could you please fix it in your branch after merging this PR

Reply:
Will do